docs/users_guide/ocean/test_groups/global_ocean.rst

```diff
@@ -305,14 +305,30 @@ ocean domain.
 A variant of this mesh has been used for low resolution simulations as part of
 the `E3SM v1 Cryosphere Campaign <https://e3sm.org/research/cryosphere-ocean/v1-cryosphere-ocean/>`_.
 
+.. _global_ocean_mesh_kuroshio:
+
+Kuroshio8to60 and Kuroshio12to60
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Kuroshio 8- and 12- to 60-km meshes (Kuroshio8to60 and Kuroshio12to60) are
+designed to explore dynamics of the Kuroshio Current.
+
+The meshes have 8 and 12 km resolution, respectively, in the eastern
+North Pacific, tapering to 60 km at mid latitudes, 30 km at the equator, and
+35 km in polar regions (the same as the EC30to60).
+
+.. image:: images/kuroshio12to60.png
```

Comment:
It the image (see below on the "Files changed" tab) okay or would you rather have something "fancier" from your simulation results?

Reply:
@xylar The image looks good, but I think this one is the Kuroshio 8km to 60km grid figure so the file name of png should be changed. I have another figure (provided by Mat) for the Kuroshio 8km to 60km grid figure, this might be "fancier" but the  first image might be better to explore the whole region.
![kuroshio8to60gridLen](https://user-images.githubusercontent.com/47384280/217941942-eb9a9731-d417-49fa-a1f0-eb91dc05a8c5.jpeg)

Reply:
Oh, sorry, I think you're right.  I can fix that.

Reply:
```suggestion
.. image:: images/kuroshio8to60.png
```

Reply:
Your (Mat's) image is better.  I'll use it.